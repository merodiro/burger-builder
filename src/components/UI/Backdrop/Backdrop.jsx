import React from 'react'
import classes from './Backdrop.module.css'

const Backdrop = ({ show, clicked }) =>
  show ? <div className={classes.Backdrop} onClick={clicked} /> : null

export default Backdrop
