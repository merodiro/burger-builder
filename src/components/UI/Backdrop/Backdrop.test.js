import React from 'react'
import ReactDOM from 'react-dom'
import { shallow } from 'enzyme'

import Backdrop from './Backdrop'

describe('<Backdrop>', () => {
  it('renders the same output', () => {
    const component = shallow(<Backdrop />)

    expect(component).toMatchSnapshot()
  })

  it('renders with show property', () => {
    const component = shallow(<Backdrop show="true">Hello world</Backdrop>)

    expect(component).toMatchSnapshot()
  })
})
