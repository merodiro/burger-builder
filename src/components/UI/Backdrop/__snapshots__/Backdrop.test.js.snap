// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Backdrop> renders the same output 1`] = `""`;

exports[`<Backdrop> renders with show property 1`] = `
<div
  className="Backdrop"
/>
`;
